packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -237,6 +237,11 @@ export type TableProps<
    * @beta
    */
   enableVirtualization?: boolean;
+  /**
+   * When using virtualized table, initialize the table scrolled to this item
+   * Does not support non-virtualized tables or tables with pagination
+   */
+  scrollToItem?: T;
```

Comment:
hm... Would we add support for all table types later on? Would it make sense?

Cause otherwise, not sure I am a fan of prop, which works only with virtualization is enabled :/